tests/e2e/tkgs_ha_utils.go

```diff
@@ -898,3 +898,17 @@ func verifyStsVolumeMetadata(client clientset.Interface, ctx context.Context, na
 	}
 
 }
+
+// verifyAnnotationsAndNodeAffinityInSVC verifies annotations on SVC PVC
+// and node affinities and pod location of volumes on correct zones
+func verifyAnnotationsAndNodeAffinityInSVC(allowedTopologyHAMap map[string][]string,
+	pod *v1.Pod, nodeList *v1.NodeList, pv *v1.PersistentVolume) {
```

Comment:
I think we can use verifyAnnotationsAndNodeAffinity method itself and add an if block for stretchedSVC if required.

Reply:
updated. Removed the new code